.circleci/config.yml

```diff
@@ -189,6 +189,16 @@ commands:
             wget https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-2.4.0.tar.gz
             sudo tar -C /usr/local -xzf ./libtensorflow-cpu-linux-x86_64-2.4.0.tar.gz
             sudo /sbin/ldconfig -v
+  install_libsodium:
+    steps:
+      - run:
+          name: Install libsodium
+          command: |
+            wget --secure-protocol=TLSv1_2 https://download.libsodium.org/libsodium/releases/libsodium-1.0.18.tar.gz
+            sudo tar -C . -xzf ./libsodium-1.0.18.tar.gz
+            cd libsodium-1.0.18 && sudo ./configure && sudo make && sudo make check && sudo make install
```

Comment:
don't need and also should not call follow commands with sudo permission :
`sudo ./configure`
`sudo make`
`sudo make check`